jupyter_matlab_proxy/app.py

```diff
@@ -398,4 +399,18 @@ def main():
         runner, host=app["settings"]["host_interface"], port=app["settings"]["app_port"]
     )
     loop.run_until_complete(site.start())
-    loop.run_forever()
+    
+    fu = loop.create_future()
```

Comment:
Another option is to keep the `loop.run_forever()`, add the signal handler before it, and then call `loop.stop()` from `shutdown`

Reply:
I must use a synchronous method for the signal handler and shutdown func must remain async so it would wait for app.shutdown() and app.cleanup().
so the lambda now calls loop.stop and then I call loop.run_until_complete(shutdown())